CONTRIBUTING.md

```diff
@@ -5,12 +5,13 @@
 
 2. **Code formatting** rules are described by the [Clang-Format Style Options] file [.clang-format].
    Please use the [clang-format] tool to format your code _changes_ accordingly.
```

Comment:
Maybe recommend a version of clang-format.

With the version available with Ubuntu Artful, I get
```
$ git-clang-format
YAML:36:23: error: unknown key 'SplitEmptyFunction'
  SplitEmptyFunction: false
                      ^~~~~
Error reading /home/yh/src/cpp-ethereum/.clang-format: Invalid argument
$ clang-format --version
clang-format version 4.0.1-6 (tags/RELEASE_401/final)
```

This can be done on a different PR.

Reply:
Mine is 6.0.0

But I don't know which one to recommend as minimum

Reply:
4.0.1 didn't work and 5.0.0 worked.